src/optional.coffee

```diff
@@ -28,7 +30,7 @@ None = {
 }
 
 toOption = (v) ->
-  if v instanceof Some or v == None
+  if v?._isSome or v == None
```

Comment:
Yes, that's a good point. It should indeed. I'll take an `ag` for more direct object comparisons.